src/common-ssh/common-ssh/ssh.h

```diff
@@ -98,7 +98,7 @@ void guac_common_ssh_uninit();
  *     if the connection or authentication were not successful.
  */
 guac_common_ssh_session* guac_common_ssh_create_session(guac_client* client,
-        const char* hostname, const char* port, guac_common_ssh_user* user);
+        const char* hostname, const char* port, guac_common_ssh_user* user, const int keepalive);
```

Comment:
As of 711cdd6, the prototype for `guac_common_ssh_create_session()` no longer matches the function definition (`const int keepalive` vs. `int keepalive`).